package.json

```diff
@@ -1,81 +1,85 @@
 {
-    "name": "rin",
-    "version": "0.0.3",
-    "description": "simple multi purpose bot",
-    "main": "lib/main.js",
-    "engines": {
-        "node": "8.9.1",
-        "npm": "5.7.1"
-    },
-    "scripts": {
-        "build": "rm lib/ -rf && ./node_modules/.bin/babel src -d lib",
-        "dev": "./node_modules/.bin/babel-watch src/main.js",
-        "cli": "npm run build && STATUS=debug npm start",
-        "start": "node lib/main.js",
-        "lint": "./node_modules/.bin/eslint --fix .",
-        "coala": "coala -n",
-        "test": "jest",
-        "heroku-postbuild": "npm run build"
-    },
-    "jest": {
-        "coverageDirectory": "./coverage/",
-        "collectCoverage": true
-    },
-    "author": "indmind",
-    "license": "MIT",
-    "dependencies": {
-        "axios": "^0.18.0",
-        "babel-core": "^6.26.0",
-        "babel-preset-env": "^1.6.1",
-        "cssbeautify": "^0.3.1",
-        "currency-formatter": "^1.4.2",
-        "discord.js": "^11.3.2",
-        "dotenv": "^5.0.1",
-        "express": "^4.16.3",
-        "figlet": "^1.2.0",
-        "goo.gl": "^0.1.4",
-        "google-translate-api": "^2.3.0",
-        "is-json": "^2.0.1",
-        "is-reachable": "^2.4.0",
-        "jawaskrip": "^0.3.4",
-        "mathjs": "^4.1.0",
-        "moment": "^2.22.0",
-        "money": "^0.2.0",
-        "node-sass": "^4.8.3",
-        "prettyjson": "^1.2.1",
-        "rata": "0.0.1",
-        "showdown": "^1.8.6",
-        "simple-markdown": "^0.3.2",
-        "telegraf": "^3.20.0",
-        "typescript-compiler": "^1.4.1",
-        "urban-dictionary": "^1.0.6",
-        "weather-js": "^2.0.0",
-        "webshot": "^0.18.0",
-        "wikijs": "^4.6.1",
-        "winston": "^2.4.1"
-    },
-    "devDependencies": {
-        "babel-cli": "^6.26.0",
-        "babel-eslint": "^8.2.2",
-        "babel-jest": "^22.4.3",
-        "babel-plugin-wildcard": "^4.0.0",
-        "babel-watch": "^2.0.7",
-        "eslint": "^4.19.1",
-        "eslint-config-prettier": "^2.9.0",
-        "eslint-plugin-import": "^2.10.0",
-        "eslint-plugin-jest": "^21.15.0",
-        "eslint-plugin-prettier": "^2.6.0",
-        "jest": "^22.4.3",
-        "prettier": "^1.11.1",
-        "regenerator-runtime": "^0.11.1"
-    },
-    "directories": {
-        "lib": "lib",
-        "view": "view"
-    },
-    "keywords": [
-        "bot",
-        "discord",
-        "telegram"
-    ]
+  "name": "rin",
```

Comment:
This file can be reformatted by sorting keys and following indentation.

*Origin: JSONFormatBear, Section: `all.json`.*

The issue can be fixed by applying the following patch:

```diff
--- a/tmp/tmp4o1f296t/package.json
+++ b/tmp/tmp4o1f296t/package.json
@@ -1,85 +1,85 @@
 {
-  "name": "rin",
-  "version": "0.0.3",
-  "description": "simple multi purpose bot",
-  "main": "lib/main.js",
-  "engines": {
-    "node": "8.9.1",
-    "npm": "5.7.1"
-  },
-  "scripts": {
-    "build": "rm lib/ -rf && ./node_modules/.bin/babel src -d lib",
-    "dev": "./node_modules/.bin/babel-watch src/main.js",
-    "cli": "npm run build && STATUS=debug npm start",
-    "start": "node lib/main.js",
-    "lint": "./node_modules/.bin/eslint --fix .",
-    "coala": "coala -n",
-    "test": "jest",
-    "heroku-postbuild": "npm run build",
-    "snyk-protect": "snyk protect",
-    "prepare": "npm run snyk-protect"
-  },
-  "jest": {
-    "coverageDirectory": "./coverage/",
-    "collectCoverage": true
-  },
-  "author": "indmind",
-  "license": "MIT",
-  "dependencies": {
-    "axios": "^0.18.0",
-    "babel-core": "^6.26.0",
-    "babel-preset-env": "^1.6.1",
-    "cssbeautify": "^0.3.1",
-    "currency-formatter": "^1.4.2",
-    "discord.js": "^11.3.2",
-    "dotenv": "^5.0.1",
-    "express": "^4.16.3",
-    "figlet": "^1.2.0",
-    "goo.gl": "^0.1.4",
-    "google-translate-api": "^2.3.0",
-    "is-json": "^2.0.1",
-    "is-reachable": "^2.4.0",
-    "jawaskrip": "^0.3.4",
-    "mathjs": "^4.1.0",
-    "moment": "^2.22.0",
-    "money": "^0.2.0",
-    "node-sass": "^4.8.3",
-    "prettyjson": "^1.2.1",
-    "rata": "0.0.1",
-    "showdown": "^1.8.6",
-    "simple-markdown": "^0.3.2",
-    "telegraf": "^3.20.0",
-    "typescript-compiler": "^1.4.1",
-    "urban-dictionary": "^1.0.6",
-    "weather-js": "^2.0.0",
-    "webshot": "^0.18.0",
-    "wikijs": "^4.6.1",
-    "winston": "^2.4.1",
-    "snyk": "^1.71.0"
-  },
-  "devDependencies": {
-    "babel-cli": "^6.26.0",
-    "babel-eslint": "^8.2.2",
-    "babel-jest": "^22.4.3",
-    "babel-plugin-wildcard": "^4.0.0",
-    "babel-watch": "^2.0.7",
-    "eslint": "^4.19.1",
-    "eslint-config-prettier": "^2.9.0",
-    "eslint-plugin-import": "^2.10.0",
-    "eslint-plugin-jest": "^21.15.0",
-    "eslint-plugin-prettier": "^2.6.0",
-    "jest": "^22.4.3",
-    "prettier": "^1.11.1",
-    "regenerator-runtime": "^0.11.1"
-  },
-  "directories": {
-    "lib": "lib",
-    "view": "view"
-  },
-  "keywords": [
-    "bot",
-    "discord",
-    "telegram"
-  ],
-  "snyk": true
+    "name": "rin",
+    "version": "0.0.3",
+    "description": "simple multi purpose bot",
+    "main": "lib/main.js",
+    "engines": {
+        "node": "8.9.1",
+        "npm": "5.7.1"
+    },
+    "scripts": {
+        "build": "rm lib/ -rf && ./node_modules/.bin/babel src -d lib",
+        "dev": "./node_modules/.bin/babel-watch src/main.js",
+        "cli": "npm run build && STATUS=debug npm start",
+        "start": "node lib/main.js",
+        "lint": "./node_modules/.bin/eslint --fix .",
+        "coala": "coala -n",
+        "test": "jest",
+        "heroku-postbuild": "npm run build",
+        "snyk-protect": "snyk protect",
+        "prepare": "npm run snyk-protect"
+    },
+    "jest": {
+        "coverageDirectory": "./coverage/",
+        "collectCoverage": true
+    },
+    "author": "indmind",
+    "license": "MIT",
+    "dependencies": {
+        "axios": "^0.18.0",
+        "babel-core": "^6.26.0",
+        "babel-preset-env": "^1.6.1",
+        "cssbeautify": "^0.3.1",
+        "currency-formatter": "^1.4.2",
+        "discord.js": "^11.3.2",
+        "dotenv": "^5.0.1",
+        "express": "^4.16.3",
+        "figlet": "^1.2.0",
+        "goo.gl": "^0.1.4",
+        "google-translate-api": "^2.3.0",
+        "is-json": "^2.0.1",
+        "is-reachable": "^2.4.0",
+        "jawaskrip": "^0.3.4",
+        "mathjs": "^4.1.0",
+        "moment": "^2.22.0",
+        "money": "^0.2.0",
+        "node-sass": "^4.8.3",
+        "prettyjson": "^1.2.1",
+        "rata": "0.0.1",
+        "showdown": "^1.8.6",
+        "simple-markdown": "^0.3.2",
+        "telegraf": "^3.20.0",
+        "typescript-compiler": "^1.4.1",
+        "urban-dictionary": "^1.0.6",
+        "weather-js": "^2.0.0",
+        "webshot": "^0.18.0",
+        "wikijs": "^4.6.1",
+        "winston": "^2.4.1",
+        "snyk": "^1.71.0"
+    },
+    "devDependencies": {
+        "babel-cli": "^6.26.0",
+        "babel-eslint": "^8.2.2",
+        "babel-jest": "^22.4.3",
+        "babel-plugin-wildcard": "^4.0.0",
+        "babel-watch": "^2.0.7",
+        "eslint": "^4.19.1",
+        "eslint-config-prettier": "^2.9.0",
+        "eslint-plugin-import": "^2.10.0",
+        "eslint-plugin-jest": "^21.15.0",
+        "eslint-plugin-prettier": "^2.6.0",
+        "jest": "^22.4.3",
+        "prettier": "^1.11.1",
+        "regenerator-runtime": "^0.11.1"
+    },
+    "directories": {
+        "lib": "lib",
+        "view": "view"
+    },
+    "keywords": [
+        "bot",
+        "discord",
+        "telegram"
+    ],
+    "snyk": true
 }
```